install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -494,10 +494,12 @@ spec:
                   path: server.crt
                 - key: tls.key
                   path: server.key
+              optional: true
```

Comment:
nit: it's easier to read/parse when `optional` is just below `name` rather than the `items` list.

Reply:
Many of the Cilium yaml definitions seem to use the alphabetical order. That's why optional is where it is.